packages/acceptance-tests/pkg-tests-specs/sources/commands/version/apply.test.ts

```diff
@@ -16,13 +16,13 @@ describe(`Commands`, () => {
         async ({path, run}) => {
           await run(`version`, `patch`, `--deferred`);
 
-          await expect(xfs.readJsonPromise(ppath.join(path, `package.json`))).resolves.toMatchObject({
+          await expect(xfs.readJsonPromise(ppath.join(path, `package.json` as PortablePath))).resolves.toMatchObject({
```

Comment:
Note that you can use `Filename.manifest` to have a properly-typed `package.json` string